addons/stock_delivery/views/delivery_view.xml

```diff
@@ -22,6 +22,7 @@
                         <field name="is_return_picking" invisible="1"/>
                         <field name="carrier_id" readonly="state in ('done', 'cancel')" options="{'no_create': True, 'no_open': True}"/>
                         <field name="delivery_type" invisible="True"/>
+                        <field name="carrier_tracking_url" invisible="True"/>
```

Comment:
Since odoo/odoo#137031, it is not needed to add invisible fields. If they are used somewhere in the template, they will be automatically added as invisible field
```suggestion
```

Reply:
done